docs/admin/system-information.rst

```diff
@@ -1574,49 +1574,161 @@ Tables need to be recreated
+    |    "temperature" REAL,                              |
+    |    PRIMARY KEY ("id")                               |
+    | )                                                   |
+    | CLUSTERED BY ("id") INTO 4 SHARDS                   |
+    | WITH (                                              |
+    |    "allocation.max_retries" = 5,                    |
+    |    "blocks.metadata" = false,                       |
+    |    "blocks.read" = false,                           |
+    |    "blocks.read_only" = false,                      |
+    |    "blocks.read_only_allow_delete" = false,         |
+    |    "blocks.write" = false,                          |
+    |    codec = 'default',                               |
+    |    column_policy = 'strict',                        |
+    |    "mapping.total_fields.limit" = 1000,             |
+    |    max_ngram_diff = 1,                              |
+    |    max_shingle_diff = 3,                            |
+    |    number_of_replicas = '0-1',                      |
+    |    refresh_interval = 1000,                         |
+    |    "routing.allocation.enable" = 'all',             |
+    |    "routing.allocation.total_shards_per_node" = -1, |
+    |    "store.type" = 'fs',                             |
+    |    "translog.durability" = 'REQUEST',               |
+    |    "translog.flush_threshold_size" = 536870912,     |
+    |    "translog.sync_interval" = 5000,                 |
+    |    "unassigned.node_left.delayed_timeout" = 60000,  |
+    |    "warmer.enabled" = true,                         |
+    |    "write.wait_for_active_shards" = '1'             |
+    | )                                                   |
+    +-----------------------------------------------------+
+    SHOW 1 row in set (... sec)
+
+2. Create a new temporary table, using the schema returned from
+   :ref:`ref-show-create-table`::
+
+    cr> CREATE TABLE rx.tmp_metrics (id TEXT PRIMARY KEY, temperature REAL);
+    CREATE OK, 1 row affected (... sec)
+
+3. Copy the data::
+
+    cr> INSERT INTO rx.tmp_metrics (id, temperature) (SELECT id, temperature FROM rx.metrics);
+    INSERT OK, 2 rows affected (... sec)
+
+4. Swap the tables::
+
+    cr> ALTER CLUSTER SWAP TABLE rx.tmp_metrics TO rx.metrics;
+    ALTER OK, 1 row affected  (... sec)
+
+5. Confirm the new ``your_table`` contains all data and has the new version::
+
+    cr> SELECT count(*) FROM rx.metrics;
+    +----------+
+    | count(*) |
+    +----------+
+    |        2 |
+    +----------+
+    SELECT 1 row in set (... sec)
 
-5) Swap the tables::
+    cr> SELECT version['created'] FROM information_schema.tables 
+    ... WHERE table_schema = 'rx' AND table_name = 'metrics';
+    +--------------------+
+    | version['created'] |
+    +--------------------+
+    | 4.2.0              |
+    +--------------------+
+    SELECT 1 row in set (... sec)
 
-    ALTER CLUSTER SWAP TABLE tmp_your_table TO your_table;
+6. Drop the now obsolete old table::
```

Comment:
```suggestion
6. Drop the old table, as it is now obsolete::
```

("now obsolete" is a bit confusing as an adjective phrase as it is used here, especially when combined with the additional adjective, "old". if you do want to keep it this way, it should be styled "now-obsolete" to disambiguate the adjectives)

Reply:
some related reading here: https://english.stackexchange.com/a/461189 (fun fact: this is one element of typographic style I'm fascinated with and have done far too much research on than I care to admit)